web/docs/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ else()
 
     set(CUSTOM_OUTPUTS)
     set(CUSTOM_INPUTS)
-    foreach (NAME triangle redball)
+    foreach (NAME triangle redball suzanne)
```

Comment:
In the future, maybe we could read the list of tutorial names in from a file.
```
file(STRINGS TUTORIALS tutorials.txt)
```
will read the contents of `tutorials.txt` into the variable `TUTORIALS`. This way, both Python and Cmake can share the list.

Reply:
Done.